.github/workflows/TriggerIntegrationTests.sh

```diff
@@ -0,0 +1,32 @@
+token=$1
+commit=$2
+repository=$3
+prNumber=$4
+frombranch=$5
+tobranch=$6
+
+getPayLoad() {
+    cat <<EOF
+{
+    "event_type": "K8sLintActionPR", 
+    "client_payload": 
+    {
+        "action": "K8sLint", 
+        "commit": "$commit", 
+        "repository": "$repository", 
+        "prNumber": "$prNumber", 
+        "tobranch": "$tobranch", 
+        "frombranch": "$frombranch"
+    }
+}
+EOF
+}
+
+response=$(curl -X POST -H "Authorization: token $token" https://api.github.com/repos/sundargs2000/azure-actions-integration-tests/dispatches --data "$(getPayLoad)")
```

Comment:
Edit it to  response=$(curl -u $patUser:$token -X POST https://api.github.com/repos/Azure/azure-actions-integration-tests/dispatches --data "$(getPayLoad)")